packages/web/src/components/track-overflow-modal/ConnectedMobileOverflowModal.tsx

```diff
@@ -192,7 +212,15 @@ const ConnectedMobileOverflowModal = ({
           onDeletePlaylist: isAlbum ? () => {} : () => deletePlaylist(id as ID),
           onPublishPlaylist: isAlbum
             ? () => {}
-            : () => publishPlaylist(id as ID)
+            : () => publishPlaylist(id as ID),
+          onPurchase: () =>
```

Comment:
do we actually need this? i thought we agreed that we don't need purchase album in the overflow menu. is there one in designs?

Reply:
Probably not, I just added it cause why not but I'm not opposed to removing it; not using it anywhere anyways

Reply:
Would you prefer to remove it?

Reply:
ah i see.. yeah no harm in leaving it in haha